theme/package.json

```diff
@@ -9,6 +9,8 @@
   "author": "Katarina",
   "license": "MIT",
   "dependencies": {
+    "breakpoint-sass": "^2.7.1",
+    "hamburgers": "^1.1.3",
```

Comment:
Is this hamburgers library still being used?

Reply:
yes